### PR TITLE
I335 inspections

### PR DESCRIPTION
**Pull request by romani**
Issue #335

### CODE REVIEW

**Comment by romani** [comment]
> src/main/java/com/github/checkstyle/github
 NotesBuilder.java (3)
64:  Single character alternation in RegExp
65:  Single character alternation in RegExp
66:  Single character alternation in RegExp


hmm, Idea did not fix this by autofix.
@rnveach , please review and merge this PR, I will try to address 3 left items a bit later

**Comment by romani** [comment]
one more commit , should work to satisfy inspections.

File: releasenotes-builder/src/test/java/com/github/checkstyle/templates/TemplateProcessorTest.java [hunk 1]
```diff
@@ -138,11 +139,11 @@ public void testGenerateOnlyNewModule() throws Exception {
     public void testGenerateAll() throws Exception {
         final List<String> errors = MainProcess.run(
             createNotes(
-                Arrays.asList(createReleaseNotesMessage("Title 1", "Author 1")),
-                Arrays.asList(createReleaseNotesMessage(2, "Title 2", "Author 2")),
-                Arrays.asList(createReleaseNotesMessage("Title 3", "Author 3")),
-                Arrays.asList(createReleaseNotesMessage(4, "Title 4", "Author 4")),
-                Arrays.asList(createReleaseNotesMessage("Title 5", "Author 5"))
+                Collections.singletonList(createReleaseNotesMessage("Title 1", "Author 1")),
```
Comment (rnveach) [inline releasenotes-builder/src/test/java/com/github/checkstyle/templates/TemplateProcessorTest.java]: Whats the benefit of this over the old?

Comment (rnveach) [reply releasenotes-builder/src/test/java/com/github/checkstyle/templates/TemplateProcessorTest.java]: Apparently it is minor performance and memory boost.

**Pull request merged**